### PR TITLE
Feature/Managing Resources

### PR DESCRIPTION
**Pull request by Shane-Gray394**
## Summary

[Video Overview](https://www.loom.com/share/8941657c8f7549a98eb9dd1a5187fd6b)

"As a mentor I want to be able to request new resources to help me perform my duties."

I constructed and fleshed out the Manage Resources component, including the request resource form and the table for viewing/filtering resources.

Fixes [#113](https://trello.com/c/Wt5Dw9Gn)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by kennedybryant — APPROVED** [review]
Shane, this looks great. Your code looks clean and is easy to follow. The styles you've implemented for this form will set a good example for the styling of components in the future. I noticed the the Run Lint on source error, but I believe this is a common on for PRs. I say it's ready to merge!

File: src/components/pages/ManageResources/RenderResourceManagement.js [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    <div>
+      {lg ? (
+        <Card
+          tabList={tabList}
+          activeTabKey={activeTabKey}
+          onTabChange={key => {
+            onTabChange(key);
+          }}
+          style={{
+            width: '40%',
```
Comment (lanceturbes) [inline src/components/pages/ManageResources/RenderResourceManagement.js:231]: I strongly suggest changing this value to either a larger or non percentage-based unit, such as 40rem; with window sizes under 1200px, a 40% width causes row labels to become unreadable (tucked underneath the table, itself). In testing, a width of 40rem produced roughly the same appearance as in your video demonstration.

**Review by lanceturbes — CHANGES_REQUESTED** [review]
Overall, very solid, but I noticed a small bug in manual testing with smaller window sizes still above the breakpoint causing row labels to not be displayed properly. It's a quick fix (see comment on line 231 of `RenderResourceManagement.js`), but one worth the edit.

I would also make note of the arrays at the top of your component being dummy data that should eventually be replaced. You did mention it in your video, but I would also note this in the code itself for future devs working on the project.

Other than that, you've done well getting the searching/filtering functionality in! The end result looks very professional.

File: src/components/pages/ManageResources/RenderResourceManagement.js (hunk 1 above)
Comment (Shane-Gray394) [reply src/components/pages/ManageResources/RenderResourceManagement.js:231]: Awesome, thanks Lance. I thought I had changed the value sooner, so I definitely appreciate you pointing that out.

**Review by lanceturbes — APPROVED** [review]
Great! Rows are readable at smaller screen sizes now, and code is commented for clarity where appropriate. Nothing else I would change -- looks perfect.

**Review by jimlemoine — COMMENTED** [review]
Really impressive functionality demonstrated in your video. I love that the column search actively filters by search input. One small suggestion - in RenderResourceManagement.js on lines 174 and 236 pertains is spelled "pertaians", the second a should probably be removed just for clarity sake.

**Pull request merged**